v3/src/hooks/use-max-attr-value-widths.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react"
+import { IDataSet } from "../data-model/data-set"
+import { measureText } from "./use-measure-text"
+
+interface IWidthEntry {
+  caseId: string
```

Comment:
What does caseId have to do with maximum width of value string? Seem unnecessary to me.

Reply:
I was thinking ahead to how we'll deal with change notifications. If you don't know the case that contained the longest string, then a change to any value requires re-assessing all values. With it, if the value of that case wasn't changed, then you only have to compare any changed values to the existing max value.